source/strings.tex

```diff
@@ -1749,9 +1749,10 @@
 \complexity Linear in the size of the sequence.
 
 \pnum
-\remarks Reallocation invalidates all the references, pointers, and iterators
-referring to the elements in the sequence as well as the past-the-end iterator.
-If no reallocation happens, they remain valid.
+\remarks
+Reallocation invalidates all the references, pointers, and iterators
+referring to the elements in the sequence, as well as the past-the-end iterator.
+If no reallocation takes place, they remain valid.
```

Comment:
@jensmaurer: Your comment about "front matter for `vector` reminded me that `shrink_to_fit` is also a member function of `basic_string` (as well as `deque`, which I already hit, above).